iterative/resource_task.go

```diff
@@ -230,6 +231,15 @@ func resourceTaskBuild(ctx context.Context, d *schema.ResourceData, m interface{
 		}
 	}
 
+	val := "true"
+	v["TPI_TASK"] = &(val)
```

Comment:
```suggestion
	v["TPI_TASK"] = &val
```

💅🏼 Do we really need parentheses here?

Reply:
Indeed we did not have it before. I wanted to not have to use an intermediate var and I overlooked it

Reply:
Go doesn't allow to take the address of an immutable value, but you can always use [`pointy`](https://github.com/openlyinc/pointy) instead. 🤷🏼‍♂️